build_tools/bazel_to_cmake/bazel_to_cmake_converter.py

```diff
@@ -591,20 +572,20 @@ def gentbl_cc_library(self,
                         includes=None,
                         strip_include_prefix=None,
                         test=None):
-    name_block = _convert_string_arg_block("NAME", name, quote=False)
-    tblgen_block = _convert_tblgen_block(tblgen)
-    td_file_block = _convert_td_file_block(td_file)
-    outs_block = _convert_tbl_outs_block(tbl_outs)
-
-    self.converter.body += (f"iree_tablegen_library(\n"
-                            f"{name_block}"
-                            f"{td_file_block}"
-                            f"{outs_block}"
-                            f"{tblgen_block}"
-                            f")\n\n")
+    name_block = self._convert_string_arg_block("NAME", name, quote=False)
+    tblgen_block = self._convert_tblgen_block(tblgen)
+    td_file_block = self._convert_td_file_block(td_file)
+    outs_block = self._convert_tbl_outs_block(tbl_outs)
+
+    self._converter.body += (f"iree_tablegen_library(\n"
+                             f"{name_block}"
+                             f"{td_file_block}"
+                             f"{outs_block}"
+                             f"{tblgen_block}"
+                             f")\n\n")
 
   def iree_gentbl_cc_library(self, **kwargs):
```

Comment:
There are still a lot of `iree_*` functions in here. Should we move these to the IREE config? Or is the idea that they're general enough that other repos could use them? We should probably rename them and move promote them to some separate place in that case, but this is OK for now.

Reply:
I was trying to split the difference between things that were "private to IREE" and things that would be used as part of openxla projects that depend on IREE. Things I judged in the latter category stayed central.

Reply:
Ok this is fine for now. Just thinking that as the boundary becomes clearer we should probably rename/migrate/generalize. We might also want to enable some way of including multiple configs, like one loading another or something.